esma.cmake

```diff
@@ -16,7 +16,11 @@ elseif (IS_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/@ecbuild")
 elseif (IS_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}/ecbuild@")
   list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/ecbuild@/cmake")
 else ()
-  message (FATAL_ERROR "ecbuild subdir not found")
+  if (DEFINED ENV{CMAKE_PREFIX_PATH})
+    list (APPEND CMAKE_PREFIX_PATH "$ENV{CMAKE_PREFIX_PATH}")
```

Comment:
@rmontuoro Do you intend to remove this ENV as well?

Reply:
@tclune I think he did already. Fast developer!

Reply:
Yes. Done in 2e7fa7a